src/lambda_codebase/initial_commit/bootstrap_repository/adf-bootstrap/deployment/global.yml

```diff
@@ -1118,12 +1118,29 @@ Resources:
                     ],
                     "Default": "EnableCrossAccountAccess"
                 },
-                "EnableCrossAccountAccess": {
-                    "Type": "Task",
-                    "Resource": "${EnableCrossAccountAccess.Arn}",
-                    "Next": "UpdateDeploymentPipelines",
-                    "TimeoutSeconds": 900
-                },
+                "EnableCrossAccountAccessMap": {
+                      "Type": "Map",
```

Comment:
The indentation is off, could you fix that?
Probably happened when the partition PR got merged.